google/cloud/aiplatform/training_jobs.py

```diff
@@ -5091,13 +5672,22 @@ def _run(
         model_tbt.display_name = model_display_name or self._display_name
         model_tbt.labels = model_labels or self._labels
 
+        # AutoMLVideo does not support validation, so pass in '-' if any other filter split is provided.
```

Comment:
This is a weird thing we have to do for AutoMLVideo, pending discussion with Video team.